Unable to use log messages with spaces in docker

This command works...

```
docker run --rm datalust/seqcli:latest log --message="{ReleaseName}inproject{ProjectName}successfullfor{ReleaseEnvironment}" \
--property=ReleaseName="Test" \
--property=ReleaseEnvironment="Test" \
--property=projectName="***" \
--property=Environment=PRODUCTION \
--property=Service="***" \
--level=Information \
--server=https://***/ \
--apikey=***
```

This one does not...

```
docker run --rm datalust/seqcli:latest log --message="{ReleaseName} in project {ProjectName} successfull for{ReleaseEnvironment}" \
--property=ReleaseName="Test" \
--property=ReleaseEnvironment="Test" \
--property=projectName="***" \
--property=Environment=PRODUCTION \
--property=Service="***" \
--level=Information \
--server=https://***/ \
--apikey=***
```

With the message `Unrecognized options: in, project, {ProjectName}, successfull, for, {ReleaseEnvironment}`

I've also tried using -m instead of --message, using ` instead of ", and removing quotes from properties.

Running on Ubuntu 18.04 with docker ce latest package version `5:18.09.6~3-0~ubuntu-bionic`

I also have run into the same issue. I tried numerous combinations: 

docker run --net host datalust/seqcli:latest log -m "Hello, World"
Unrecognized options: World

docker run --net host datalust/seqcli:latest log -m 'Hello, World'
Unrecognized options: World

docker run --net host datalust/seqcli:latest log -m `Hello, World`
Hello,: command not found
The command failed: Missing required value for option '-m'.

I was able to get it to work by modifying the **run.sh** file to:

`#!/bin/bash
exec /bin/seqcli/seqcli "$@"`

Note the quotes around $@ (https://github.com/koalaman/shellcheck/wiki/SC2068)

The following commands then worked:
docker run --rm --net host seqcli log -m "Hello World"
docker run --rm --net host seqcli log -m "Hello {Name}" -p Name="Linus Torvalds"

Cheers

--- a/dockerfiles/seqcli/run.sh
+++ b/dockerfiles/seqcli/run.sh
@@ -1,6 +1,2 @@
 #!/bin/bash
-
-# Default arguments to those passed to the container
-args=$@
-
-exec /bin/seqcli/seqcli $args
+exec /bin/seqcli/seqcli "$@"

